app/routes/_gcn.circulars/circulars.lib.ts

```diff
@@ -36,6 +36,20 @@ type SubjectMatcher = [RegExp, (match: RegExpMatchArray) => string]
 
 const subjectMatchers: SubjectMatcher[] = [
   [/GRB[.\s_-]*(\d{6}[a-z|.]\d*)/i, ([, id]) => `GRB ${id.toUpperCase()}`],
+  [/GRB[.\s_-]*(\d{6}[a-z|.]\d*):/i, ([, id]) => `GRB ${id.toUpperCase()}`],
```

Comment:
I think that these are duplicates; strings that match the first of these two regexes will also match the second.